.github/workflows/scripts/countline.py

```
@@ -0,0 +1,295 @@
+    result = True
+
+    # Test
+    for invalid_filename in invalid_filenames:
+        if invalid_filename.lower() not in filepath.lower():
+            continue
+        result = False
+
+    return result
+
+
+def _valid_extension(filepath):
```

Ruff (0.12.2)

Missing return type annotation for private function `_valid_extension`

(ANN202)